state/accountsDB.go

```diff
@@ -1206,6 +1211,18 @@ func (adb *AccountsDB) finishSnapshotOperation(
 	stats.PrintStats(message, rootHash)
 }
 
+func (adb *AccountsDB) updateMetricsOnSnapshotCompletion(message string, stats *snapshotStatistics) {
+	if message == peerTrieSnapshotMsg {
+		adb.appStatusHandler.SetStringValue(common.MetricPeersSnapshotIsProgress, "false")
```

Comment:
[comment]

I thought you wanted to set it using `adb.isSnapshotInProgress.IsSet()` to keep consistency. Why changed it to "false"?

Reply:
i set it directly to false since i've added separate metrics for accounts and peer, that flag is not relevant to be used to both